install/lib/class.installer.php

```diff
@@ -380,6 +392,40 @@ private static function __checkConfiguration()
         }
 
         /**
+         * This function checks if there is a unattend.php file in the MANIFEST folder.
+         * If it finds one, it will load it and check for the $settings variable.
+         * It will also merge the default config values into the 'fields' array.
+         *
+         * You can find an empty version at install/include/unattend.php
+         *
+         * @return array
+         *   An associative array of values, as if it was submitted by a POST
+         */
+        private static function __checkUnattended()
```

Comment:
I know, but I tried to kept things as they currently are. There will be a major refactor of this soon ;)